apps/system/js/nfc_handover_manager.js

```diff
@@ -361,10 +311,10 @@
      * @memberof NfcHandoverManager.prototype
      */
     _doConnect: function _doConnect(device) {
-      this._debug('doConnect with: ' + device.address);
-      var req = this.defaultAdapter.connect(device);
-      req.onsuccess = () => { this._debug('Connect succeeded'); };
-      req.onerror = () => { this._debug('Connect failed'); };
+      this.debug('doConnect with: ' + device.address);
+      var req = this._adapter.connect(device);
```

Comment:
Got it! I'm okay in this version. But keep in mind the adapter is possible removed if out platform is ready to support plug-in Bluetooth device in some day.